src/chocolatey/infrastructure.app/commands/ChocolateyUpgradeCommand.cs

```diff
@@ -264,7 +265,7 @@ public virtual void Validate(ChocolateyConfiguration configuration)
             if (!string.IsNullOrWhiteSpace(configuration.Input))
             {
                 var unparsedOptionsAndPackages = configuration.Input.Split(new[] { " " }, StringSplitOptions.RemoveEmptyEntries);
-                if (!configuration.Information.IsLicensedVersion)
+                if (!(configuration.Information.IsLicensedVersion && configuration.Information.IsLicensedAssemblyLoaded))

```

Comment:
Same as above for the `install` command.

Reply:
As per our discussion on Zoom, I've backed out this change.